src/RegistryImportFeed.php

```diff
@@ -117,16 +115,16 @@ public function getStoragePath($modelClass = null)
     public function getImportFiles()
     {
         $path = $this->getStoragePath();
-        $importFiles = $this->getAssetHandler()->getFilesystem()->listFiles($path);
+        $importFiles = $this->getAssetHandler()->getFilesystem()->listContents($path)->toArray();
 
         $files = ArrayList::create();
 
         foreach ($importFiles as $importFile) {
             $files->push(RegistryImportFeedEntry::create(
-                $importFile['basename'],
+                basename($importFile->path()),
                 '',
-                DBDatetime::create()->setValue($importFile['timestamp'])->Format(DBDatetime::ISO_DATETIME),
-                $this->getAssetsDir() . '/' . $importFile['path']
+                DBDatetime::create()->setValue($importFile->lastModified())->Format(DBDatetime::ISO_DATETIME),
+                $this->getAssetsDir() . '/' . $importFile->path()
```

Comment:
What are all of these changes doing? Why are these necessary? They don't seem relevant to removing a call to `addPlugin()`?
Or is this replacing the functionality that plugin was applying?

Reply:
All this changes have been done to support new version of `league/flysystem`. I've added test especially for this method.